lib/Bailador/Request.pm

```diff
@@ -25,40 +27,61 @@ class Bailador::Request {
     }
     multi method params ($source) {
         my %ret;
+        my $source_data = '';
```

Comment:
Because in line 42 (`return %ret unless $source_data.chars;`), $source_data could still be uninitialized, and checking `$source_data.chars` is issuing a warning. I'm not sure how to deal with that other than initializing the variable:

```
 perl6 -e 'my $src; say 42 unless $src.chars'
Method 'chars' not found for invocant of class 'Any'
  in block <unit> at -e line 1
```

```
$ perl6 -e 'my Str $src; say 42 unless $src.chars'
Use of uninitialized value of type Str in string context
Any of .^name, .perl, .gist, or .say can stringify undefined things, if needed.  in block <unit> at -e line 1
42
```

```
$ perl6 -e 'my $src = q<>; say 42 unless $src.chars'
42
```

If there's a more idiomatic way to write this bit in Perl 6, I'd love to hear about it and amend the PR :)